packages/connector/src/types.ts

```diff
@@ -67,6 +69,16 @@ export type SaveCredentialsRequestParams = {
   type: string;
 };
 
+export type RemoveCredentialsRequestParams =
+  | {
+      ids: string[];
```

Comment:
What ids? Consider giving a more descriptive name

Reply:
@ihordiachenko should we explicitly say `credentialIds` if we remove credentials?

Reply:
If we keep both credential IDs and claim IDs - yes, because now I find it confusing